simcon_project/conversation_templates/models/template_response.py

```diff
@@ -1,10 +1,15 @@
 from django.db import models
+
 import uuid
 
 
-class TemplateResponse (models.Model):
+class TemplateResponse(models.Model):
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     completion_date = models.DateTimeField(default=None)
     student = models.ForeignKey('users.Student', related_name='template_responses', default=0, on_delete=models.CASCADE)
-    template = models.ForeignKey('conversation_templates.ConversationTemplate', default=0, related_name='template_responses', on_delete=models.CASCADE)
-    assignment = models.ForeignKey('users.Assignment', default=0, related_name='template_responses', on_delete=models.CASCADE)
+    template = models.ForeignKey('conversation_templates.ConversationTemplate', default=0,
```

Comment:
Can you add this to the formatting page on Confluence so every file is formatted this way if we are choosing this option! Thankyou